java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -206,7 +206,9 @@ public enum OrcConf {
       "orc.proleptic.gregorian.default", false,
       "This value controls whether pre-ORC 27 files are using the hybrid or proleptic\n" +
       "calendar. Only Hive 3.1 and the C++ library wrote using the proleptic, so hybrid\n" +
-      "is the default.")
+      "is the default."),
+  STRIPE_ROW_COUNT("orc.stripe.row.count","hive.exec.orc.stripe.row.count",
```

Comment:
If hive doesn't have this configuration, to avoid being misleading, I think we should set hiveConfName to be the same as attribute.

Reply:
fixed